src/biotite/sequence/graphics/alignment.py

```diff
@@ -344,7 +344,142 @@ def get_color(self, alignment, column_i, seq_i):
         code = alignment.sequences[seq_i].code[index]
         return self._colors[code]
 
+    
+class ArrayPlotter(LetterPlotter):
+    '''
+    This SymbolPlotter quantitatively decorates sequences alignments, with molecular
+    recognition data obtaine from microarrays. Symbols are visualized as characters 
+    on a colored background box. The color of a given box represents the recognition 
+    signal. The intensity of the color, is proportional to the strenght of the 
+    recognition.
+    
+    Parameters
+    ----------
+    axes : Axes
+        A Matplotlib axes, that is used as plotting area.
+    fl_score : numpy.ndarray 
+        The ndarray to map recognition values to score residues.
+        By default the normalized score is 1 for maximun recognition
+        and 0 for non-recognition(no color).
+    color_symbols : bool, optional
+        If true, the symbols themselves are colored.
+        If false, the symbols are black, and the boxes behind the
+        symbols are colored.
+    font_size : float, optional
+        Font size of the sequence symbols.
+    font_param : dict, optional
+        Additional parameters that is given to the
+        matplotlib.Text instance of each symbol.
+    
+    '''
+    def __init__(self, axes, fl_score, color_symbols=False,
+                 font_size=None, font_param=None):
+
+        super().__init__(axes, color_symbols, font_size, font_param)
+        if fl_score is not None:
+            self.fl_score = fl_score 
+        else:
+            self.fl_score = None        
```

Comment:
Does it make sense to use an array plotter without values for the score? Nevertheless, this code can be shortened:
```suggestion
        self.fl_score = fl_score       
```